src/pages/sunday-school-page.tsx

```diff
@@ -1,7 +1,35 @@
 export const SundaySchool = () => {
   return (
     <div>
-      <h1>Sunday School</h1>
-    </div>
+      <body>
+        <div id="SundaySchoolSummary">
+          <h1 id="SundaySchoolHeader">Sunday School</h1>
+          <p>Let's learn Arabic, Islamic studies, and Quran at Sunday School</p>
+        </div>
+        <div id="TeacherInfoSection">
+          <h2 id="TeacherInfoHeader">Teacher Information</h2>
+          <img id="TeacherImage" src="teacherimage.jpg"/>
+          <p id="TeacherBio">Ms. Valerie Frizzle is a phenomenal teacher that has been working within the program for 20 years</p>
```

Comment:
Is this stuff being pulled from the admin page? I just did filler text for now. This question applies to most other things on the page as well

Reply:
Yeah...we'll get dummy text into the view and then come back and clean it up once the back-end gets crystallized

Reply:
How far should I take this skeleton then? Just add some more dummy text and call it a day or mess around with the layout a bit more without the CSS? I guess I'm not sure what state to get this to for the time being, and I know it will be getting another pass later on down the line anyhow.

Reply:
tbh the styling is really what I'm after here.  I'm not sure I have a good idea of what a standard page should look like, so I'm just kinda fucking around with ideas.  If you wanna stop here, that is _more_ than ok, but if not...feel free to get creative!  You can take a look at the little I did on the services page (I just merged it if you wanna pull `main`).